modules/administration-guide/pages/running-containers-with-kubedock.adoc

```diff
@@ -0,0 +1,66 @@
+:_content-type: ASSEMBLY
+:description: Running containers with kubedock
+:keywords: kubedock, container
+:navtitle: Running containers with kubedock
+:page-aliases:
+
+[id="running-containers-with-kubedock"]
+= Running containers with kubedock
+
+link:https://github.com/joyrex2001/[Kubedock] is a minimal container engine implementation that gives you a Podman-/docker-like experience inside a {prod-short} workspace. Kubedock is especially useful when dealing with ad-hoc, ephemeral, and testing containers, such as in the use cases listed below:
```

Comment:
The correct link is this:
https://github.com/joyrex2001/kubedock

Reply:
fixed